### ISSUE TITLE
Revise the color of the page background on content pages to pass accessibility contrast test 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/lenape-timetree#107]
@jhimpele @rlskoeser When designing the content pages our homepage background color that you currently see on the site is not passing the WCAG AAA test (for reading smaller text on the page, against our text color). 
Therefore I had to revise our colors a little. [Please look at the new colors on the content page](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1047%3A5827&t=WfUVCcrtNoS3ym1v-1) – [you can also see what it would have looked like, which failed the accessibility test](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1047%3A6350&t=WfUVCcrtNoS3ym1v-1)

I have two proposals, we can choose one from below: 

- We can revise the page background color everywhere on the site (including the homepage) so that it's a bit darker, and revise the header/footer/leaf panel colors too so that there is still a contrast between the two colors. 
![Screenshot 2023-03-14 at 6.06.27 PM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/5f741505-a2c4-4e71-af2f-3f4d5c03a24c)

- Or we can only revise the page background color on the content pages (root pages) and leave the homepage as is. But we will still need to change the header/footer/leaf panel colors so that they match everywhere on the site. 
![Screenshot 2023-03-14 at 6.06.49 PM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/a398da09-8ff6-4e02-8e58-12621fa806bc)

**Comment by jhimpele** [comment]
@gissoo I'm sorry, I'm having a hard time following what is the old, and what are the new options. It looks like you are proposing a darker background for the content pages, which has less contrast with the header/footer. But we may or may not have to darken the main tree page background. Is that right? If so, I prefer to make the content pages dark, but keep the tree page lighter....  @rlskoeser what do you think.

**Comment by gissoo** [comment]
@rlskoeser @jhimpele based on our discussions yesterday [I have revised and have two proposals here on Figma](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1085%3A3069&t=grrfKlJsCLU1VgPf-1) 
- in proposal 1 I have a new background color we could apply to all pages that's warmer (based on your feedback), and I have also revised the header and footer and leaf panel colors to have more contrast against this background color. (and made the drop shadow of the panel starker too)
- in proposal 2 is similar to proposal 1 but the only difference is I am keeping the homepage's background color as it was before.

Which one do you like more?
I just don't really like the amount of contrast between the two colors on the homepage in proposal 2.
  
I'd like to also not invert the colors as I am seeing it will need more adjustments to text styles for accessibility, and it conceptually doesn't make sense to me with the roots section being lighter than the body.

**Comment by rlskoeser** [comment]
@gissoo thank you for finding a warmer color and looking at our options! I prefer proposal 2 but I'd be ok with proposal 1. 

There's enough contrast for me on proposal 1, not sure the drop shadow is necessary.

**Comment by gissoo** [comment]
@jhimpele I think anything lighter than the new background color will be quite similar to what we had before – since neither of you feel strongly about this and are open about either option I propose we go with proposal 1 since it's the safest for accessibility. And since we like the glow for the plaque which would work on this darker background. 

I'm still not convinced that the drop shadow on the leaf labels on the lighter background is enough for accessibility, I don't really have a way to test it on my end.

**Comment by jhimpele** [comment]
Sounds good. I’d avoid the drop shadow if we can….


Jeffrey Himpele.
Director, VizE Lab for Ethnographic Data Visualization
Lecturer in Anthropology
vizelab.princeton.edu<http://vizelab.princeton.edu/>


From: gissoo ***@***.***>
Date: Wednesday, March 22, 2023 at 2:01 PM
To: Princeton-CDH/lenape-timetree ***@***.***>
Cc: Jeffrey D. Himpele ***@***.***>, Mention ***@***.***>
Subject: Re: [Princeton-CDH/lenape-timetree] Revise the color of the page background on content pages to pass accessibility contrast test (Issue #107)

@jhimpele<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjhimpele&data=05%7C01%7Cjhimpele%40princeton.edu%7C90859005d8d8474bdcbc08db2aff85a2%7C2ff601167431425db5af077d7791bda4%7C0%7C0%7C638151049169652084%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6upVoTC9OY9sOdZ2eCIH%2F6jhnksu9XqYsHJuOqDJVn8%3D&reserved=0> I think anything lighter than the new background color will be quite similar to what we had before – since neither of you feel strongly about this and are open about either option I propose we go with proposal 1 since it's the safest for accessibility. And since we like the glow for the plaque which would work on this darker background.

I'm still not convinced that the drop shadow on the leaf labels on the lighter background is enough for accessibility, I don't really have a way to test it on my end.
